tests/modules/bowtie2/align/test.yml

```diff
@@ -8,6 +8,16 @@
     - path: ./output/bowtie2/test.bowtie2.log
     - path: ./output/bowtie2/versions.yml
 
+- name: bowtie2 align test_bowtie2_align_single_end_sorted
+  command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_single_end_sorted -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
+  tags:
+    - bowtie2
+    - bowtie2/align
+  files:
+    - path: ./output/bowtie2/test.bam
+    - path: ./output/bowtie2/test.bowtie2.log
+    - path: ./output/bowtie2/versions.yml
+
```

Comment:
Maybe also adding a test for paired-end data as in the [bwa module](https://github.com/nf-core/modules/blob/919275103a00ddbcfc31b2c424837efaaf2ef0a8/tests/modules/bwa/mem/main.nf#L60-L72)

Reply:
done!